.azure-pipelines/ipu-tests.yml

```diff
@@ -90,6 +90,7 @@ jobs:
     - bash: |
         source ${{ variables.poplar_sdk }}/poplar-ubuntu*/enable.sh
         source ${{ variables.poplar_sdk }}/popart-ubuntu*/enable.sh
+        export POPTORCH_WAIT_FOR_IPU=1
```

Comment:
@awaelchli What does this do?

Reply:
https://docs.graphcore.ai/projects/poptorch-user-guide/en/latest/overview.html?highlight=POPTORCH_WAIT_FOR_IPU#wait-for-an-ipu-to-become-available

It is waiting instead crashing when all IPUs are in use but we are requesting more.

Reply:
as best as I can tell, it has no influence on the runtime in the CI, but I was running into issues 
`Failed to acquire 1 IPU(s) Context: LowerToPopart::compile`
and had to add it.